.gitignore

```diff
@@ -3,6 +3,12 @@ pids
 *.pid
 *.seed
 
+# Directory for instrumented libs generated by jscoverage/JSCover
```

Comment:
Yeah I think you can leave those out - they're automatically generated by github when you create a repo with a pre-built 'Node' `.gitignore` - the template source is [here](https://github.com/github/gitignore/blob/master/Node.gitignore).